src/flint/fmpz_poly.jl

```diff
@@ -582,8 +582,6 @@ function interpolate(R::ZZPolyRing, x::Vector{ZZRingElem},
   ax = Vector{Int}(undef, length(x))
```

Comment:
Ah so here we have the same problematic interface and you just copied it. OK, fair enough -- but it would still be better to have the split I described. Would you perhaps be willing to quickly do that (it shouldn't be too much work I think?)

But I also understand if that's not something  you are interested in, in that case we can take care of it ourselves later.